python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -2769,6 +2769,42 @@ def Aten_EmbeddingBagExample_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class CumsumModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, val):
+        return torch.ops.aten.cumsum(val, 1)
+
+@register_test_case(module_factory=lambda: CumsumModule())
+def CumsumModule_basic(module, tu: TestUtils):
+    module.forward(torch.randn(2, 7, 4))
```

Comment:
nit: use `tu.rand` to generate random tensors. See: https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#test-case-function. Same comment applies to the other test